### PR TITLE
feat: add a11y contribution type

### PR DESCRIPTION
**Pull request by phacks**
**What**: Add `a11y` (Accessibility) contribution type.

**Why**: Discussed in https://github.com/all-contributors/all-contributors/issues/320

**How**: Added a new contribution type in `src/util/contribution-types.js`

**Checklist**:

- [x] Documentation: in another PR
- [x] Tests: I don’t think tests are needed for this PR, please tell me if you’d like some
- [x] Ready to be merged 
- [ ] Added myself to contributors table 

I’ve yet to figure out how to add myself to the contributors table, can I use the @all-contributors bot to this end?

### CODE REVIEW

File: src/util/contribution-types.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ const repo = require('../repo')
 
 const defaultTypes = function(repoType) {
   return {
+    a11y: {
+      symbol: '️️️️♿️',
+      description: 'Accessibility',
+    }
```
Comment (gr2m) [inline src/util/contribution-types.js]: Forgot a comma :)

```suggestion
    },
```

**Comment by Berkmann18** [comment]
> I’ve yet to figure out how to add myself to the contributors table, can I use the @all-contributors bot to this end?

Don't worry about that part, we can always do that as this PR gets merged.

File: src/util/contribution-types.js (hunk 1 above)
Comment (gr2m) [reply src/util/contribution-types.js]: I don't have the permission to fix it myself. @phacks can you or someone with write access get this fix commited so we can merge the PR? I've already merged the PR for the website: https://github.com/all-contributors/all-contributors/pull/326

Comment (Berkmann18) [reply src/util/contribution-types.js]: I tried fixing this but I don't have permission either, that being said I didn't try to fix that locally.

Comment (phacks) [reply src/util/contribution-types.js]: oops 🙈 I corrected this in 485e598, sorry about that!

**Comment by phacks** [comment]
@Berkmann18 Hi! I think this PR can be merged, now that I fixed the typo and created https://github.com/all-contributors/all-contributors-bot/pull/259, what do you think?

**Comment by Berkmann18** [comment]
@all-contributors Please add @phacks for code

**Pull request merged**